.github/workflows/ci.yml

```
@@ -21,22 +21,19 @@ jobs:
         id: release-please
-        with:
-          app-id: ${{ vars.RELEASE_PLEASE_GITHUB_APP_ID }}
-          private-key: ${{ secrets.RELEASE_PLEASE_GITHUB_APP_PRIVATE_KEY }}
 
   release:
     runs-on: ubuntu-latest
     needs: [lint, release-please]
-    if: needs.release-please.outputs.release_created
+    if: ${{ needs.release-please.outputs.release_created }}
     steps:
       - uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)